addons/im_livechat/security/im_livechat_channel_security.xml

```diff
@@ -20,6 +20,13 @@
             <field name="users" eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"/>
         </record>
 
+        <record id="im_livechat_rule_manager_read_all_mail_channel" model="ir.rule">
+            <field name="name">Livechat: Administrator: read all channel</field>
+            <field name="model_id" ref="model_mail_channel"/>
+            <field name="groups" eval="[(4, ref('im_livechat_group_manager'))]"/>
+            <field name="domain_force">[(1, '=', 1)]</field>
```

Comment:
If I'm not mistaken, that will allow the livechat administrator to actually read ALL channels (and not only the livechat ones).
Even, for example, private conversations between two employees.

Seems a little bit dangerous.